bots/csr/src/main/java/org/openjdk/skara/bots/csr/CSRBot.java

```diff
@@ -153,6 +177,12 @@ public Collection<WorkItem> run(Path scratchPath) {
                 log.info("CSR closed and approved for " + describe(pr) + ", removing CSR label");
                 pr.removeLabel(CSR_LABEL);
             }
+            if (!hasCsrIssueAndProgressChecked(pr, csr)) {
+                // If the PR body doesn't have the CSR issue or doesn't have the CSR progress or the CSR progress checkbox is not selected,
+                // this bot need to add the csr update marker so that the PR bot can update the message of the PR body.
+                log.info("CSR closed and approved for " + describe(pr) + ", adding the csr update marker");
+                pr.setBody(pr.body() + CSR_UPDATE_MARKER);
```

Comment:
We need to make sure we only add the update marker once. Otherwise we would end up adding a very large amount of these if the PR bot happens to be stuck for a while. I would recommend a method for adding the marker, which first checks if the body contains the marker already.